src/domain/Phone/WebRTCPhone.js

```diff
@@ -207,6 +215,12 @@ export default class WebRTCPhone extends Emitter implements Phone {
     this.client.changeAudioOutputDevice(id);
   }
 
+  changeRingDevice(id: string) {
+    this.audioRingDeviceId = id;
+    // @TODO: make sure this is ok
+    // this.client.changeAudioOutputDevice(id);
```

Comment:
Still todo ?

Reply:
well... if by "still" you mean "is it _still_ as work in progress", i'd say yeah :)
At this point, i don't have a complete grasp of what the client does in relation to the phone class, just wasn't sure this was OK, so i figured i'll leave a few breadcrumbs for now. 

Wait a minute -- shouldn't you be on vacation right now?

Reply:
issues don't take a vacation